docs/lib/storage/fragments/flutter/download.md

```diff
@@ -10,3 +10,20 @@ try {
   print(e.toString());
 }
 ```
+
+### Generate a download URL
```

Comment:
Does this belong in this pull?

Reply:
Yes one additional update to storage we discussed

Reply:
yes we wanted a minor update to our Storage docs to include GetURL